docs/releases.md

```diff
@@ -49,7 +49,19 @@ locally.
 Create a PR from your branch, this will then run through the tests for the new
 release. If it's green, you can merge it into master.
 
-## 4. Create the Github release
+## 4. Re-tag after the PR is merged
+
+It is encouraged to re-tag the code once the PR has been merged to keep a clean
+commit log. To do so, once the PR has been merged:
+
+```bash
+git checkout master
+git pull origin master
+cat VERSION # ensure VERSION has the correct tag
+inv git.tag --foce
```

Comment:
Typo `force`

Reply:
amazed at the fact that i write five lines, one of them has a typo, and i don't see it :eyeglasses: